lib/Gitlab/Api/Groups.php

```diff
@@ -18,7 +20,7 @@ class Groups extends AbstractApi
     public function all(array $parameters = [])
     {
         $resolver = $this->createOptionsResolver();
-        $booleanNormalizer = function ($value) {
+        $booleanNormalizer = function (OptionsResolver $resolver, $value) {
```

Comment:
Could you typehint with `Symfony\Component\OptionsResolver\Options` instead?

Reply:
Didn't realize there's an interface... done (see amended commit)